governance.md

```diff
@@ -4,46 +4,43 @@ The official version of this document, along with a list of
-developed openly and hosted in public GitHub repositories under the
-[IPython GitHub organization](https://github.com/ipython) and the
-[Jupyter GitHub organization](https://github.com/jupyter). Examples of
-Project Software include the IPython Notebook, the IPython Terminal,
-IPython.parallel, the Jupyter Hub, etc. The Services run by the
-Project consist of public websites and web-services that are hosted
-under the jupyter.org or ipython.org domains. The Services Examples of
-Project Services include the Jupyter and IPython websites
-([http://jupyter.org](http://jupyter.org) and
-[http://ipython.org](http://ipython.org)), nbviewer
-([https://nbviewer.ipython.org](https://nbviewer.ipython.org)) and the
-Jupyter coLaboratory
-([https://colaboratory.jupyter.org](https://colaboratory.jupyter.org)).
+The Matplotlib Project (The Project) is an open source software
+project affiliated with the 501c3 NumFocus Foundation. The goal of The
+Project is to develop open source software and deploy open and public
+websites and services for data visualization. The Software developed
+by The Project is released under the BSD (or similar permissive) open
+source license, developed openly and hosted in public GitHub
+repositories under the
+[Matplotlib GitHub organization](https://github.com/matplotlib). Examples
+of Project Software include the matplotlib library for data
+visualization and associated extensions and dependencies.  The
+Services run by the Project consist of public websites and
+web-services that are hosted under the matplotlib.org
+domain.
 
 The Project is developed by a team of distributed developers, called
 Contributors. Contributors are individuals who have contributed code,
-documentation, designs or other work to one or more Project repositories.
-Anyone can be a Contributor. Contributors can be affiliated with any legal
-entity or none. Contributors participate in the project by submitting,
-reviewing and discussing GitHub Pull Requests and Issues and participating in
-open and public Project discussions on GitHub, Google+, Hackpad, Gitter chat
-rooms and mailing lists. The foundation of Project participation is openness
-and transparency.
+documentation, designs or other work to one or more Project
+repositories.  Contributors are also known by the name "Matplotlib
+Development Team (MDT)" in the project license.  Anyone can be a
+Contributor. Contributors can be affiliated with any legal entity or
+none. Contributors participate in the project by submitting, reviewing
+and discussing GitHub Pull Requests and Issues and participating in
+open and public Project discussions on GitHub, Google+, Hackpad,
+Gitter chat rooms and mailing lists. The foundation of Project
+participation is openness and transparency.
 
-Here is a list of the current Contributors to the main IPython repository:
+Here is a list of the current Contributors to the main Matplotlib repository:
```

Comment:
I have had some discussion with a jupyter core member (@Carreau) about the limitations of 'contributor' in this document.